README.md

```diff
@@ -113,6 +113,11 @@ You can also instruct a writer to use line buffering, instead of the usual chunk
 Finally, you can control the character(s) used to indicate comments by setting the
 `comment` keyword when writing a file. By default, there is no character (""). During reading, the comment character is found atomatically.
 
+Note that, by default, these reader functions will assume UTF-8 encoding. You can choose a
+different character encoding by setting the `encoding` keyword argument to any of these
+reader or writer functions. For example, on Windows, Python will assume Windows-1252 encoding,
+which can be specified via `encoding='cp1252'`.

```

Comment:
This line here might be a bit confusing for the reader. If Python assumes Windows 1252 encoding, then what's the default, UTF-8 or cp1252? I know what the code does, but it won't be clear for a reader if they need to use the cp1252 encoding or not.

Reply:
Yeah, sorry, I should have been more clear. 

I guess the case in which this is important is if a Windows user has written a csv file using Python but outside of pycsvy, which by default would use cp1252-encoded. If they don't pay/haven't paid attention to encodings then it could cause confusion for them.